### PR TITLE
Add installation instructions from OSX issue

### LINKED ISSUE #77: Add note that gsl and fftw can be installed via conda
As @binnietom as suggested, both gsl and fftw can be installed with `conda`, which should make the installation process even easier for those using conda. 

This should be added to the installation instructions. We should also consider creating a conda package so that the entire installation can proceed in one line.

### PR DESCRIPTION
**Pull request by steven-murray**
Closes #77 .

### CODE REVIEW

File: INSTALLATION.rst [hunk 1]
```diff
@@ -2,11 +2,62 @@
...
+MacOSX
+~~~~~~
+On MacOSX, obtaining `gsl` and `fftw` is typically more difficult, and in addition,
+the newer native `clang` does not offer `-fopenmp` support.
+
+For `conda` users (which we recommend using), the easiest way to get `gsl` and `fftw`
+is by doing `conda install -c conda-forge gsl fftw` in your environment.
+
+To get `gcc`, either use `homebrew`, or again, `conda`: `conda install -c anaconda gcc`.
+If you get the `conda` version, you still need to install the headers::
```
Comment (dprelogo) [inline INSTALLATION.rst]: Maybe just to add a comment here (or in Compile Options) that for conda installed libs, the easiest command is just:
`CC=/path/to/gcc LIB=/path/to/conda_environment/lib INC=/path/to/conda_environment/include pip install ... `
as libs are all in the same folder in conda. Then all the problems one might have is to get a  "good" `gcc`

Comment (steven-murray) [reply INSTALLATION.rst]: Ahhh, yes good point. I'll add that in.

Comment (steven-murray) [reply INSTALLATION.rst]: Fixed in  75fbc29

**Comment by steven-murray** [comment]
@dprelogo this is ready to review again :-)

**Pull request merged**